hydroshare/settings.py

```diff
@@ -856,7 +859,3 @@ def _(s): return s # noqa
 )
 
 BULK_UPDATE_CREATE_BATCH_SIZE = 1000
-
-if ENABLE_OIDC_AUTHENTICATION:
```

Comment:
This block was not working as intended and never resolves to True. This means that the configuration in this block is unnecessary.

Reply:
so you can remove this `HydroShareOIDCAuthenticationBackend` class?
https://github.com/hydroshare/hydroshare/blob/2.13/hs_core/authentication.py#L9-L23

Reply:
We deifnitely need that create_user method override. I guess it is actually working.  The only reference to that class on www is in settings.py and it looks like the if block does successfully modify the AUTHENTICATION_BACKENDS list as intended. I'm not sure what I was seeing before.

```python
>>> from django.conf import settings
>>> settings.AUTHENTICATION_BACKENDS
['mezzanine.core.auth_backends.MezzanineBackend', 'theme.backends.CaseInsensitiveMezzanineBackend', 'hs_core.authentication.HydroShareOIDCAuthenticationBackend']
```
```bash
[hydro-service@www1-hydroshare hydroshare]$ git grep 'hs_core.authentication.HydroShareOIDCAuthenticationBackend'
hydroshare/settings.py:    AUTHENTICATION_BACKENDS.append("hs_core.authentication.HydroShareOIDCAuthenticationBackend")
```

Reply:
wild. Okay I'm not sure what I was looking at either...